assets/src/edit-story/elements/image/save.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import getPercentageFromPixels from '../../utils/getPercentageFromPixels';
+
+/**
+ * Returns AMP HTML for saving into post content for displaying in the FE.
+ */
+function ImageSave( { id, src, width, height, x, y, rotationAngle } ) {
+	const props = {
+		layout: 'fill',
+		src,
+	};
+	const wrapperProps = {
+		id: 'el-' + id,
+	};
+	const style = {
+		position: 'absolute',
+		left: getPercentageFromPixels( x, 'x' ) + '%',
+		top: getPercentageFromPixels( y, 'y' ) + '%',
+		width: getPercentageFromPixels( width, 'x' ) + '%',
+		height: getPercentageFromPixels( height, 'y' ) + '%',
+		transform: `rotate(${ rotationAngle }deg)`,
```

Comment:
Not the focus of this PR, but just quick nuance: `transform: rotate(0deg)` has a lot of performance and other nuances. Generally it'd be safer to not set `transform` at all if no transformation is needed.

Reply:
Improved in 5469f09.

Reply:
Just a tiny comment on this w.r.t. `null` vs `''` style values. Not 100% this is still a case today, but it used to be pretty riddled with browser issues.

Reply:
Hmm, I was certain that React would omit that style then, however, will check with @barklund

Reply:
I think you're right. At least in this case, since we are writing only once, I don't think it actually matters. Let's proceed.

Reply:
Sounds good!